.github/workflows/build.yml

```
@@ -73,11 +73,11 @@ jobs:
           RUST_BACKTRACE: 1
 
       - name: cargo build (debug; no default features)
-        run: cargo build --no-default-features
+        run: cargo test --no-default-features
         working-directory: rustls
 
-      - name: cargo test (debug; no default features)
-        run: cargo test --no-default-features
+      - name: cargo test (debug; without logging feature)
```

small nit: Would this name be clearer as "(debug; no default features; tls1.2)"? It seems slightly more future proof :-)

```suggestion
      - name: cargo test (debug; no default features; tls1.2)
```

---

Thanks, I think that's also clearer. I squashed all the ci commits in https://github.com/rustls/rustls/pull/1213/commits/8d79b1aa7365dce42e3c74eba3e0aead94925bce

---

Looks great, thanks!